### PR TITLE
Avatar component

### PR DESCRIPTION
**Pull request by prachi-sahani**
Created avatar component
Types of avatars included:
 - basic avatar with image
 - avatar with icon
 - avatars based on sizes - xs, sm, md, lg, xl
 - square avatars 
 - group of avatars(will wrap in case of overflow)
 - stacked avatars

image

### CODE REVIEW

File: css/components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+.avatar-icon{
+    background-color: var(--primary-900);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: var(--black-100);
+}
+
+/* for avatar with an image */
+.avatar-img{
```
Comment (ZuberDunge) [inline css/components/avatar.css:22]: add: object-fit :cover;

**Review by ZuberDunge — COMMENTED** [review]
**LGTM!**

File: css/components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
+@import url(../../css/utilities/color.css);
```
Comment (junaidshaikh-js) [inline css/components/avatar.css]: Give proper relative path

File: css/components/avatar.css (hunk 1 above)
Comment (prachi-sahani) [reply css/components/avatar.css:22]: Done!

File: css/components/avatar.css (hunk 2 above)
Comment (prachi-sahani) [reply css/components/avatar.css]: Done!

**Pull request merged**